transports/azure-service-bus/legacy/transaction-support.md

```diff
@@ -46,3 +46,8 @@ When transactions are disabled in NServiceBus then the transport uses the Azure
 The message is deleted from the queue directly after the receive operation completes, before it is processed, meaning that it's not possible to retry that message in case of processing failures. As transient exceptions occur regularly when integrating with online services, disabling retries when in unreliable mode is not recommended. This mode should only be used in very specific situations, when message loss is acceptable.
 
 NOTE: For a full explanation of the transactional behavior in Azure, refer to [Understanding internal transactions and delivery guarantees](understanding-transactions-and-delivery-guarantees.md).
+
+
+## Duplicate Detection
+
+Using the supported transaction levels, the transport only provides an **`at-least-once`** delivery model. Azure Service Bus provides a [Duplicate detection feature](https://docs.microsoft.com/en-us/azure/service-bus-messaging/duplicate-detection) which will ensure the message is received exactly once if enabled. However, Duplicate Detection comes at the expense of throughput and is time constrained.
```

Comment:
Thing is, our Delayed retries will block customers from using native deduplication feature. If we're adding this section here, I'd add a note about that as well.

Reply:
interesting. How does it block customers? or does the native deduplication "swallow" delayed retries?

Sounds like we should clearly state the drawbacks or even remove this section altogether too?

Reply:
> does the native deduplication "swallow" delayed retries?

Exactly that. I'm 👍 to note that as a feature conflicting with NSB's Delayed Retries.